From 54c95ca8f76f412c94a9f61e3756420ac554310b Mon Sep 17 00:00:00 2001
From: Ong Boon Leong <boon.leong.ong@intel.com>
Date: Thu, 12 Feb 2015 16:33:49 +0800
Subject: [PATCH] examples: add config variable to enable/disable dpdk_qat

Upstream-Status: Inappropriate [configuration]

This patch adss following example build configs:

1) CONFIG_EXAMPLE_DPDK_QAT is used to control dpdk_qat example
   by PACKAGECONFIG.

2) CONFIG_EXAMPLE_VM_POWER_MANAGER is used to control vm_power_manager
   example by PACKAGECONFIG.

Signed-off-by: Ong Boon Leong <boon.leong.ong@intel.com>
---
 examples/Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/examples/Makefile b/examples/Makefile
index 81f1d2f..7e9c7bd 100644
--- a/examples/Makefile
+++ b/examples/Makefile
@@ -40,7 +40,7 @@ include $(RTE_SDK)/mk/rte.vars.mk
 DIRS-y += cmdline
 DIRS-$(CONFIG_RTE_LIBRTE_DISTRIBUTOR) += distributor
 ifneq ($(ICP_ROOT),)
-DIRS-y += dpdk_qat
+DIRS-$(CONFIG_EXAMPLE_DPDK_QAT) += dpdk_qat
 endif
 DIRS-y += exception_path
 DIRS-y += helloworld
@@ -68,6 +68,6 @@ DIRS-$(CONFIG_RTE_LIBRTE_VHOST) += vhost
 DIRS-$(CONFIG_RTE_LIBRTE_XEN_DOM0) += vhost_xen
 DIRS-y += vmdq
 DIRS-y += vmdq_dcb
-DIRS-y += vm_power_manager
+DIRS-$(CONFIG_EXAMPLE_VM_POWER_MANAGER) += vm_power_manager
 
 include $(RTE_SDK)/mk/rte.extsubdir.mk
-- 
1.7.9.5

